ui.R

```diff
@@ -104,41 +129,37 @@ dashboardPage(
                 "Filter on number of free throws attempted:",
                 min = 0,
                 max = 881,
+                step = 5,
                 value = c(0, 881)
               )
             ),
             img(src = "Giannis.png", height = 219, width = 300, align = "middle")
           ),
-
+          
           # Column two displays the Histogram of the distrubition of the free throw attempts
           column(
             8,
-            h1("Histogram"),
-
+            h2("Histogram"),
+            
             plotOutput("histogramNBA"),
             # Add rollover for Histogram of Free Throw Proportion Plot
             bsPopover(id = "histogramNBA", title = " ", content = "This histogram filters the NBA players based on games played or free throw attempts. Changing the slider will adjust the number of players that fit the selected criteria.", placement = "left", trigger = "hover", options = NULL)
```

Comment:
Let's think about moving this text out of a popover and as a paragraph element above everything. The main point of the text feels much more like what the user should do on this page.